### PR TITLE
fix broken tests

### PR DESCRIPTION
**Pull request by manishtomar**
There turned out to be 2 response variables due to last lint change. Now renamed the argument passed to `expected_code`.

### CODE REVIEW

File: autoscale_cloudroast/test_repo/autoscale/system/schedule_policies/test_system_multi_scheduler_webhook_policies.py [hunk 1]
```diff
@@ -160,12 +160,12 @@ def _create_multi_policy_group(self, multi_num, response, *args):
...
-        response = self.autoscale_behaviors.create_scaling_group_given(
+        create_response = self.autoscale_behaviors.create_scaling_group_given(
             lc_name='multi_scheduling',
             sp_list=policy_list,
             gc_cooldown=0)
         self.assertEquals(
-            response.status_code, response,
+            create_response.status_code, response,
             msg=('Creating multiple scaling policies within a group failed '
                  'with response code: {0}'.format(response.status_code)))
```
Comment (cyli) [inline autoscale_cloudroast/test_repo/autoscale/system/schedule_policies/test_system_multi_scheduler_webhook_policies.py]: Should this also be formatted to `create_response.status_code`?

**Comment by cyli** [comment]
Other the comment above, LGTM :)

**Comment by manishtomar** [comment]
Sorry @cyli , the above code is broken. I had fixed it locally and did not commit. Don't know whats wrong with me.

**Comment by cyli** [comment]
@manishtomar bleh, I should have looked at the surrounding code - thought it took a response, sorry.  Thanks!  That makes more sense.

**Comment by cyli** [comment]
+1

**Pull request merged**